profiles/build.profile.js

```diff
@@ -0,0 +1,65 @@
+/*eslint-disable no-unused-vars*/
+var profile = {
+    basePath: '../src',
+    action: 'release',
+    cssOptimize: 'comments',
+    mini: true,
+    optimize: 'uglify',
+    layerOptimize: 'uglify',
+    stripConsole: 'all',
+    selectorEngine: 'acme',
+    layers: {
+        'dojo/dojo': {
+            include: [
+                'dojo/i18n',
+                'dojo/domReady',
+                'app/packages',
+                'app/run',
+                'app/App',
+                'dojox/gfx/path',
+                'dojox/gfx/svg',
+                'dojox/gfx/shape'
+            ],
+            targetStylesheet: 'app/resources/App.css',
+            includeLocales: ['en-us'],
+            customBase: true,
+            boot: true
+        },
+        'ijit/widgets/authentication/UserAdmin': {
+            exclude: ['dojo/dojo']
+        }
+    },
+    packages: [{
```

Comment:
The `packages` property can be removed since you aren't defining any. Looks like `resourceTags` is a relic form the `proj4` package.